testdata/runtests.bash

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+
+set -o errexit -o noclobber -o nounset -o pipefail
```

Comment:
The errexit option is causing the build step to fail. Don't know why.

Reply:
OK, I'll try re-introducing your changes slowly to figure out where it breaks.